packages/cloudflare-workers-bindings-extension/package.json

```diff
@@ -38,6 +38,12 @@
 				"icon": "$(add)"
 			}
 		],
+		"jsonValidation": [
+			{
+				"fileMatch": "**/wrangler.{json,jsonc,toml}",
+				"url": "https://www.unpkg.com/wrangler/config-schema.json"
```

Comment:
question: could unpkg version being different from wrangler:workspace be an issue?

Reply:
Potentially—which is why we also bundle the schema with Wrangler, and include a schema comment in generated config files from C3 to reference the locally installed schema directly. This is just a fallback to make sure people who don't have a schema comment get _something_, even if it's not perfect.

Reply:
Thanks - valuable info to have somewhere in the PR.